client/my-sites/plugins/main.jsx

```diff
@@ -484,40 +481,34 @@ export class PluginsMain extends Component {
 				) }
 				{ this.renderPageViewTracking() }
 				{ ! isJetpackCloud && (
-					<FixedNavigationHeader
-						className="plugin__header"
-						compactBreadcrumb={ false }
-						navigationItems={ this.props.breadcrumbs }
-					/>
+					<NavigationHeader
+						navigationItems={ [] }
+						title={ pageTitle }
+						subtitle={
+							this.props.selectedSite
+								? this.props.translate( 'Manage all plugins installed on %(selectedSite)s', {
+										args: {
+											selectedSite: this.props.selectedSite.domain,
+										},
+								  } )
+								: this.props.translate( 'Manage plugins installed on all sites' )
+						}
+					>
+						{ ! isJetpackCloud && (
+							<>
+								{ this.renderAddPluginButton() }
+								{ this.renderUploadPluginButton() }
+								<UpdatePlugins isWpCom plugins={ currentPlugins } />
+							</>
+						) }
+					</NavigationHeader>
 				) }
 				<div
 					className={ classNames( 'plugins__top-container', {
 						'plugins__top-container-wp': ! isJetpackCloud,
 					} ) }
 				>
 					<div className="plugins__content-wrapper">
-						<div className="plugins__page-title-container">
-							<div className="plugins__header-left-content">
-								<h2 className="plugins__page-title">{ pageTitle }</h2>
-								<div className="plugins__page-subtitle">
-									{ this.props.selectedSite
-										? this.props.translate( 'Manage all plugins installed on %(selectedSite)s', {
-												args: {
-													selectedSite: this.props.selectedSite.domain,
-												},
-										  } )
-										: this.props.translate( 'Manage plugins installed on all sites' ) }
-								</div>
-							</div>
```

Comment:
<img width="1712" alt="Screenshot 2023-10-24 at 11 43 25 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/96e81e55-6907-4b67-812f-f3a31db13e5c">

I don't see the heading in Jetpack Cloud, may be it is because of this.

Reply:
I'm fixing via https://github.com/Automattic/jetpack-manage/issues/60

Reply:
Fixed via https://github.com/Automattic/wp-calypso/pull/83396